source/set_up_3dsecure/index.html.md.erb

```diff
@@ -15,6 +15,8 @@ If you only take payments within the UK, you must implement SCA by 14 September
 
 To implement SCA, you must set up 3D Secure 2 (3DS2) or 3D Secure 1 (3DS1) payment authentication.
 
+You must implement SCA regardless of whether you [integrate with the GOV.UK Pay API](/integrate_with_govuk_pay/#integrate-with-the-gov-uk-pay-api) or use [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/).
```

Comment:
Again, not sure it's necessary to call out when they are the same

Reply:
ok